org.eclipse.lemminx/src/main/java/org/eclipse/lemminx/extensions/xinclude/XIncludeErrorCode.java

```diff
@@ -0,0 +1,135 @@
+/**
+ *  Copyright (c) 2018 Angelo ZERR
+ *  All rights reserved. This program and the accompanying materials
+ *  are made available under the terms of the Eclipse Public License v2.0
+ *  which accompanies this distribution, and is available at
+ *  http://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ *  Contributors:
+ *  Angelo Zerr <angelo.zerr@gmail.com> - initial API and implementation
+ */
+package org.eclipse.lemminx.extensions.xinclude;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.apache.xerces.xni.XMLLocator;
+import org.eclipse.lemminx.dom.DOMDocument;
+import org.eclipse.lemminx.services.extensions.diagnostics.IXMLErrorCode;
+import org.eclipse.lemminx.utils.XMLPositionUtility;
+import org.eclipse.lsp4j.Range;
+
+/**
+ * XInclude error code.
+ *
+ * @see https://wiki.xmldation.com/Support/Validator
+ *
+ *      All error code types and messages can be found in the Xerces library
+ *      https://github.com/apache/xerces2-j/blob/trunk/src/org/apache/xerces/impl/msg/XIncludeMessages.properties
+ *
+ */
+public enum XIncludeErrorCode implements IXMLErrorCode {
+
+	NoFallback("NoFallback"), //
```

Comment:
remove all String constant from parameter, since the error code if XINclude seems not having some special characters (like '-').

In other words, it should be just:

`NoFallback, // ...`